pkg/rtc/transport.go

```diff
@@ -894,6 +894,10 @@ func (t *PCTransport) getOnNegotiationFailed() func() {
 }
 
 func (t *PCTransport) Negotiate(force bool) {
+	if t.isClosed.Load() {
+		return
+	}
```

Comment:
Just not setting up a debounce for negotiate if the transport is closed already.